### DISCUSSION TITLE
Nuxt2 project deployment issue with node.js 18.0 to Vercel

### DISCUSSION THREAD

**Discussion by jackyyangtw** [vercel/community#3617] [Help] [answered]
### Summary

Why does my Nuxt2 project throw the error in the image when deployed to Vercel using node js 18.x engine?

Here is my project commit link:
https://github.com/jackyyangtw/Frontend-Fusion/blob/13ca3bb3b2b1eca1fc14fbacbe93324d6bbd280f/package.json

### Steps to Reproduce

1. Change the engines.node to 18.x in the package.json file.
![螢幕擷取畫面 2023-08-15 162444](https://github.com/vercel/community/assets/50864464/a53e9138-32b5-447a-9afd-97fb5d4a8220)

2. Deploying to Vercel failed.
![18 0 0](https://github.com/vercel/community/assets/50864464/11746093-853f-40f5-9e5c-8f46dc2f54db)

3. If the engines.node setting is removed, the following error will occur:
![14 x錯誤](https://github.com/vercel/community/assets/50864464/74586c2f-9a2c-43e3-bff3-1e2fd5b30590)

Here is my project commit link:
https://github.com/jackyyangtw/Frontend-Fusion/blob/13ca3bb3b2b1eca1fc14fbacbe93324d6bbd280f/package.json

**Comment by TooTallNate** [comment, marked as answer]
Logs say you are using v0.21.0 of `@nuxt/vercel-builder`. Try updating that to the latest version.
